smart-contracts/scripts/setBridgeBank.js

```diff
@@ -0,0 +1,75 @@
+
+module.exports = async (cb) => {
+    const expectedUsage = () => {console.log("Expected usage:\nBRIDGEBANK_ADDRESS='insert bridgebank address' COSMOS_BRIDGE_ADDRESS='insert cosmosbridge address' truffle exec scripts/setBridgeBank.js --network mainnet\n")}
+    try {
+      /*******************************************
+       *** Set up
+      ******************************************/
+      const ethers = require("ethers");
+  
+      const CosmosBridgeContract = artifacts.require("CosmosBridge");
```

Comment:
This is still using truffle. Move over to using hardhat build artifacts instead of truffle's.